simple_salesforce/api.py

```diff
@@ -423,6 +444,115 @@ def _call_salesforce(self, method, url, **kwargs):
 
         return result
 
+    def _call_bulk_api(self, method, url, **kwargs):
+        """Makes a call to the bulk api
+
+        Returns a `request.result` object.
+        """
+        result = self.request.request(
+            method, url, headers=self.bulk_headers, **kwargs)
+
+        if result.status_code >= 300:
+            _exception_handler(result)
+
+        return result
+
+    def _make_bulk_request(self, url, data):
```

Comment:
If this is the only entry point to `_call_bulk_api`, is there a reason to have this extra function given it's only 3 lines?